/**:
  ros__parameters:

    # from gnss
    gnss_particle_covariance:
      [
        1.0, 0.0, 0.0,  0.0,  0.0,  0.0,
        0.0, 1.0, 0.0,  0.0,  0.0,  0.0,
        0.0, 0.0, 0.01, 0.0,  0.0,  0.0,
        0.0, 0.0, 0.0,  0.01, 0.0,  0.0,
        0.0, 0.0, 0.0,  0.0,  0.01, 0.0,
        0.0, 0.0, 0.0,  0.0,  0.0,  10.0,
      ]

    # output
    output_pose_covariance:
      [
        1.0, 0.0, 0.0,  0.0,  0.0,  0.0,
        0.0, 1.0, 0.0,  0.0,  0.0,  0.0,
        0.0, 0.0, 0.01, 0.0,  0.0,  0.0,
        0.0, 0.0, 0.0,  0.01, 0.0,  0.0,
        0.0, 0.0, 0.0,  0.0,  0.01, 0.0,
        0.0, 0.0, 0.0,  0.0,  0.0,  0.2,
      ]

    # use partial map loading for large maps
    # Note: You also need to enable partial_map_loading interface in pointcloud_map_loader to use this functionality
    enable_partial_map_load: false
